cmd/agent/main.go

```diff
@@ -73,6 +73,7 @@ func main() {
 		v,
 		command,
 		flags.AddConfigFileFlag,
+		flags.AddLoggingFlag,
```

Comment:
you're certainly adding it to the cmd line, but is it being used by main to control logging?

Reply:
yup, on L45